packages/tablesorter/src/providers/JSONDataProvider.ts

```diff
@@ -34,7 +34,7 @@ import * as d3 from "d3";
 
 const debug = require("debug"); // tslint:disable-line
 const log = debug("essex:widget:tablesorter:JSONDataProvider");
-import ldIsNaN = require("lodash/isNaN");
+const ldIsNaN = require("lodash/lang/isNaN"); //tslint:disable-line
```

Comment:
Can you reference lodash internal functions using import if you install `@types/lodash`?